<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find Rack</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1921</width>
      <height>1051</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="user_prompt_txtedit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>1501</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="response_lbl">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>1511</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="querry_lbl">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>960</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 4px solid #844b36
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="browse_book_lbl">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>960</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 4px solid #844b36
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="choose_llm_lbl">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>960</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 4px solid #844b36
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="contact_us_lbl">
    <property name="geometry">
     <rect>
      <x>1510</x>
      <y>960</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 4px solid #844b36
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="choose_embedding_model_lbl">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>930</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 4px solid #844b36
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="message_lbl">
    <property name="geometry">
     <rect>
      <x>1690</x>
      <y>1010</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #a86147;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
